Tasks/GithubRelease/operations/EditRelease.ts

```diff
@@ -0,0 +1,45 @@
+import tl = require("vsts-task-lib/task");
+import { WebRequest, sendRequest, WebResponse } from "./webClient";
+import * as Utility from "./Utility";
+import { GetReleaseByTag } from "./GetReleaseByTag";
+import util = require("util");
+
+export async function editRelease(): Promise<WebResponse> {
+
+    let releaseResponse = await GetReleaseByTag();
+
+    if (releaseResponse.statusCode === 200) {
+        // Get task inputs
+        const repositoryName = tl.getInput('repositoryName');
```

Comment:
Don't take a dependency on the tl.input method in the REST helper, instead pass them as an argument to the method.  Taking dependency on tl.input in each method has the following disadvantages.

1.  It reduces reusability of the module. 
2.  In the future, if you change the name of any input then you have to change it at multiple places.